src/stories/Button.tsx

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+import "./button.css";
+
+interface ButtonProps {
+  /**
+   * Is this the principal call to action on the page?
+   */
+  primary?: boolean;
+  /**
+   * What background color to use
+   */
+  backgroundColor?: string;
+  /**
+   * How large should the button be?
+   */
+  size?: "small" | "medium" | "large";
+  /**
+   * Button contents
+   */
+  label: string;
+  /**
+   * Optional click handler
+   */
+  onClick?: () => void;
+}
+
+/**
+ * Primary UI component for user interaction
+ */
+export const Button = ({
+  primary = false,
+  size = "medium",
+  backgroundColor,
+  label,
+  ...props
+}: ButtonProps) => {
+  const mode = primary
```

Comment:
해당 삼항연산자는 template literal을 통해 중복되는 문자열은 한번만 쓸 수도 있으니 참고해보세요.

```suggestion
  const mode = `storybook-button--${primary ? 'primary : secondary}`
```

Reply:
중복되는 문자열을 피할 수 있는 방법에 대해서 미처 생각하지 못했었습니다! template literal를 통해 최소화 시키겠습니다 !!